reports/JSV.md

```diff
@@ -2,8 +2,6 @@
 
 # All validators fail this test
```

Comment:
Previously they were all failing "1.0 is not an integer" test. Somehow it is not failed by json-schema-validator-generator (that now fails almost everything else). Can you maybe rerun the tests on your machine?